README.md

```diff
@@ -132,7 +132,7 @@ component and Apache Kafka as the unified data bus.
 
 Some high level links to the relevant subparts of the architecture, for
 more information:
```

Comment:
Going forward in this effort, as it is more compassable, it may _not_ be kafka that is providing the data, should we explicitly name kafka here?

Reply:
I'm personally inclined to leave it, because that coupling still exists as of this PR.  I'm inclined to leave it as follow-on for anyone inclined to decouple our ingest from Kafka. A fair number of our examples and configs are heavily tied to Kafka right now (e.g. the parser chaining / aggregation), so I'd be more inclined to have a more thorough decoupling.

Having said that, I'd be more open to changing specific instances if there's something we feel needs to be changed now.

Reply:
Fair enough, it just stuck out to me.  Kafka is a part of the storm support, not the parsing ( we could have a udp bolt or something, or a nifi bolt ) with where we are going.